docs/reference/treyresearch/armTemplates/auxiliary/policies.json

```diff
@@ -20173,7 +20173,7 @@
                   "Disabled"
                 ],
                 "metadata": {
-                  "displayName": "Cognitive Services accounts should enable data encryption with a customer-managed key (CMK)",
+                  "displayName": "Cognitive Services accounts should enable data encryption with a customer-managed key (CMK)",
                   "description": "Customer-managed keys (CMK) are commonly required to meet regulatory compliance standards. CMKs enable the data stored in Cognitive Services to be encrypted with an Azure Key Vault key created and owned by you. You have full control and responsibility for the key lifecycle, including rotation and management. Learn more about CMK encryption at https://aka.ms/cosmosdb-cmk."
                 }
               },
```

Comment:
We are no longer using this template.

Reply:
Any harm in this fix?

Reply:
No harm, but we are planning to delete these files so I don't see any reason to why we need to update them